src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -0,0 +1,135 @@
+package org.broadinstitute.hellbender.tools.funcotator;
+
+import htsjdk.tribble.Feature;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.funcotator.dataSources.gencode.GencodeFuncotationFactory;
+import org.broadinstitute.hellbender.tools.funcotator.vcfOutput.VcfOutputRenderer;
+import org.broadinstitute.hellbender.utils.codecs.GENCODE.*;
+
+import java.io.File;
+import java.util.*;
+
+/**
+ * Funcotator (FUNCtional annOTATOR) performs functional analysis on given variants
+ * and reports output in a specified output file.
+ *
+ * This tool is the GATK analog of the Oncotator.
+ *
+ * Created by jonn on 8/22/17.
+ */
+@CommandLineProgramProperties(
+        summary = "Create functional annotations on given variants cross-referenced by a given database.\n" +
+                "A GATK version of the Oncotator.",
+        oneLineSummary = "(Experimental) Functional Annotator",
+        programGroup = VariantProgramGroup.class
+)
+@DocumentedFeature
+@BetaFeature
+public class Funcotator extends VariantWalker {
+
+    public static final String GTF_FILE_ARG_LONG_NAME = "gtfFile";
+    public static final String GTF_FILE_ARG_SHORT_NAME = "gtf";
+    public static final String GENCODE_FASTA_ARG_NAME = "fasta";
```

Comment:
Should this argument name mention gencode?

Reply:
Renamed to `gencodeFasta`